benchmark/suite.js

```diff
@@ -0,0 +1,126 @@
+import vm from 'node:vm';
+import benchmark from 'benchmark';
+import psl from '../index.js';
+
+const runSuite = ({ module, version }) => new Promise((resolve, reject) => {
+  const suite = new benchmark.Suite(version);
+
+  suite.add('psl#isValid', {
+    fn: () => {
+      module.isValid('google.com');
+    }
+  });
+
+  suite.add('psl#parse', {
+    fn: () => {
+      module.parse('google.com');
+    }
+  });
+
+  suite.add('psl#parse invalid domain', {
+    fn: () => {
+      module.parse('google.comp');
+    }
+  });
```

Comment:
I believe we should add more examples to the suite just so we're testing edge cases:

1.- Extremely long URLs
2.- Complex URLs
3.- The ones with non-conventional characters
4.- Wildcard domains?
5.- Domains with hyphens, underscores
6.- Extremely nested subdomains

Let me know what you think, I can do that and send a PR